tasks/broad/DragenTasks.wdl

```diff
@@ -42,16 +42,16 @@ task CalibrateDragstrModel {
   Int disk_size_gb = ceil(size([ref_fasta, ref_fasta_idx, alignment, str_table_file], "GiB")) + 
                         20 # 20 for the rest of the fs.
 
-  String parallel_args  = if (java_threads <= 1) then "" else "--threads " + threads
+  String parallel_args  = if (java_threads <= 1) then "" else "--threads " + java_threads
```

Comment:
Just confirming you intended to switch from threads to java_threads?

Reply:
To me it looked like we specifically calculate java_threads, but then never used the value. I think this was the intended behavior, let me know if you think otherwise